packages/store/script/Store.s.sol

```diff
@@ -0,0 +1,58 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.8.0;
+
+import "forge-std/Script.sol";
+import { StoreView } from "../src/StoreView.sol";
+import { StoreCore } from "../src/StoreCore.sol";
+import { Schema, SchemaLib } from "../src/Schema.sol";
+import { SchemaType } from "../src/Types.sol";
+
+contract ScriptStore is StoreView {
+  function registerSchema(bytes32 table, Schema schema) public override {
+    StoreCore.registerSchema(table, schema);
+  }
+
+  function setRecord(
+    bytes32 table,
+    bytes32[] memory key,
+    bytes memory data
+  ) public override {
+    StoreCore.setRecord(table, key, data);
+  }
+
+  function setField(
+    bytes32 table,
+    bytes32[] memory key,
+    uint8 schemaIndex,
+    bytes memory data
+  ) public override {
+    StoreCore.setField(table, key, schemaIndex, data);
+  }
+}
+
+contract StoreScript is Script {
+  function run() public {
+    vm.startBroadcast();
+
+    uint256 blockNumber = block.number;
```

Comment:
I've found that this isn't the actual block number that these txs are executed on. Forge seems to run this script in its entirety to grab all the txs that need executing, then executes each of them in a batch, so this will often be behind by a couple blocks.

Reply:
that's good to know!